types/jasmine-reporters/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        // Classes are used so they can implment Jasmine interfaces.
+        "no-unnecessary-class": false
```

Comment:
The rule complained about e.g. `export class AppVeyorReporter implements jasmine.CustomReporter {`

For documentation purposes I have included the Jasmine interface. AFAIK I should have a class to implement an interface.

According to the documentation of this library officially there are no additional methods / fields exposed in constructed objects.
Everything we know about those objects is that they are compatible with Jasmine.
See: https://github.com/larrymyers/jasmine-reporters#basic-usage